app/views/graphql_devise/mailer/confirmation_instructions.html.erb

```diff
@@ -1,5 +1,5 @@
 <p><%= t(:welcome).capitalize + ' ' + @email %>!</p>
 
-<p><%= t '.confirm_link_msg' %> </p>
+<p><%= t '.confirm_link_msg' %></p>
 
 <p><%= link_to t('.confirm_account_link'), url_for(controller: 'graphql_devise/graphql', action: :auth, **confirmation_query(resource_name: @resource.class.to_s, redirect_url: message['redirect-url'], token: @token)) %></p>
```

Comment:
Off topic (sort of) but we may want to implement *.text.erb counter parts to our mailer files.